themes/docs-new/layouts/partials/gtag.html

```diff
@@ -0,0 +1,27 @@
+{{ with .Site.GoogleAnalytics }}
```

Comment:
```suggestion
{{ with .Site.GoogleAnalytics }}
```
We can ditch this, and also the `{{end}}` at the bottom.